src/cosalib/aws.py

```diff
@@ -0,0 +1,19 @@
+import boto3
+from cosalib.cmdlib import (
+    retry_stop,
+    retry_boto_exception,
+    retry_callback
+)
+from tenacity import retry
+
+
+@retry(stop=retry_stop, retry=retry_boto_exception, before_sleep=retry_callback)
+def deregister_ami(ami_id, region):
```

Comment:
nit: Missing docstring

Reply:
But we're just wrapping the underlying API call...doesn't seem worth it to copy/paste it or have even more boilerplate docs like "Retry deregister_ami" right?

Feels like we want to generate these.

(On this topic of course, https://www.terraform.io/ is one of several projects implementing a "retry IaaS API calls" pattern)

Reply:
Agree, these are just simple wrappers, I don't think its worth adding docstrings there

Reply:
This may be more of a personal preference, but having a simple doc with a function tends to be helpful. Keep in mind, this is a nit and isn't required to be updated for merge. I asked @vrutkovs if he's ready for this to merge via chat as I'm happy to hit the button :slightly_smiling_face:.

What I hope for is:
```
"""
Removes an AMI from a specific region.
"""
```

or even

```
"""
.. seealso:: :py:func:`ec2.deregister_image`
"""
```

but your point is well taken @cgwalters.